src/hooks/aelin/useAelinCreatePool.tsx

```diff
@@ -156,6 +161,19 @@ export const createPoolConfig: Record<CreatePoolSteps, CreatePoolStepInfo> = {
 
 export const createPoolConfigArr = Object.values(createPoolConfig)
 
+const getWhiteListAmount = (
+  amount: number,
+  whiteListAmountFormat: AddressesWhiteListAmountFormat,
+  investmentTokenDecimals: number,
+): BigNumberish => {
+  switch (whiteListAmountFormat) {
+    case AddressesWhiteListAmountFormat.decimal:
+      return parseUnits(amount.toString(), investmentTokenDecimals).toString()
+    case AddressesWhiteListAmountFormat.uint256:
+      return amount
+  }
```

Comment:
Maybe add a default value that throws an error?

Reply:
I'm not sure that we need it because rn switch is exhaustive and there will be compilation error otherwise